### ISSUE TITLE
Linux RubyMine existing stable installation, now ruby-debug-ide fails install

### ISSUE THREAD

**Issue by JenniferMB** [ruby-debug/ruby-debug-ide#177]
Error states Unable to resolve dependency: user requested 'debase (>0)'

Problem began after performing gem housekeeping activities, removing outdated versions of gems. I have attempted gem installations with several different versions of debase and ruby-debug-ide, both stable and beta, I even updated RubyMine to 2019.2, always the same error.

Re-installation options for products like Ruby, RubyGems, Bundler, rspec, are limited due to mandatory system isolation constraints, so activities like that are not performed unless there is reason to suspect an issue with an individual product. Even the process of obtaining different gem versions and the RubyMine IDE update are laborious.

Debug functions up to this point have been reliable.

**Comment by ViugiNick** [comment]
@JenniferMB Could you please attach `gem list`?

**Comment by JenniferMB** [comment]
Unfortunately, I can't attach a gem list, without typing each individual one, and I have over 150 gems, I'd really rather not. Can I just list the ones I think may be relevant to my scenario?

- --Rails 5.2.3
- --Ruby 2.6.2
- debase (0.3.0.beta3, 0.2.3.beta5, 0.2.3)
- debase-ruby_core_source (0.10.5)

Any others that might be impacting my ruby-debug-ide installation? I have downloaded and attempted to install these:

- ruby-debug-ide-0.7.0.beta4.gem
- ruby-debug-ide-0.7.0.beta7.gem
- ruby-debug-ide-0.7.0.gem
- ruby-debug-ide-0.8.0.beta3.gem

Every time, I get the same error: 
```
...
/home/user/.rbenv/versions/2.6.2/bin/ruby  mkrf_conf.rb
Installing base gem 
Unable to resolve dependency: user requested 'debase(>0)'
/home/user/.rbenv/versions/2.6.2/bin/ruby/2.6.0/rubygems/resolver/installer_set.rb:72:in 'add_always_install'
/home/user/.rbenv/versions/2.6.2/bin/ruby/2.6.0/rubygems/resolver/dependency_installer.rb:478:in 'resolve_dependencies'
/home/user/.rbenv/versions/2.6.2/bin/ruby/2.6.0/rubygems/resolver/dependency_installer.rb:385:in 'install'
...
```

**Comment by JenniferMB** [comment]
Clearly (to me, at least), the issue is related to this block in mkrf_conf.rb
```
if RUBY_VERSION < "1.9"
 dep = Gem::Dependency.new("ruby-debug-base", '>=0.10.4')
elsif RUBY_VERSION < '2.0'
 dep = Gem::Dependency.new("ruby-debug-base19x", '>=0.111.30.pre15')
else
 dep = Gem::Dependency.new("debase", '> 0'
end
```
but I don't know what to do about it (sad face)

**Comment by JenniferMB** [comment]
Is there a way to force install the gem despite the dependency?

**Comment by ViugiNick** [comment]
@JenniferMB Maybe updating rubygems will help (not sure about it). 
https://rubygems.org/gems/rubygems-update

**Comment by JenniferMB** [comment]
You think updating RubyGems might fix the problem that the ruby-debug-ide can't identify the existence of the debase gem in my updated RubyMine installation?
Okay, I will give it a try. Thanks for responding

**Comment by JenniferMB** [comment]
Sadly, updating RubyGems had no positive effect on the installation of ruby-debug-ide. Is there a way I can manipulate the dependency options of this gem to modify or bypass the requirement?

**Comment by JenniferMB** [comment]
I was able to obtain permission to access public internet from my development computer for a few minutes, and gems installed without further issue. Lesson learned here is that offline install does not work, an active internet connection is mandatory.

**Comment by ViugiNick** [comment]
@JenniferMB Oh, you didn't mention that you are working offline. If so, then I have a fix for such problems https://github.com/ruby-debug/ruby-debug-ide/pull/142. It will be fixed in `ruby-debug-ide-0.8.0.beta6`, which will be bundled in RM 2019.2.1 RC

**Comment by JenniferMB** [comment]
That is excellent news, because it is very difficult to obtain permission
for internet access. I look forward to the RM update.
